### PR TITLE
add dhrubesh's pin to the map

### PR DESCRIPTION
**Pull request by dhrubesh**
adding India to the map!

### CODE REVIEW

File: _pins/dhrubesh.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: dhrubesh
+lattitude: 13.082680
```
Comment (brianamarie) [inline _pins/dhrubesh.json]: Hi @dhrubesh, it looks like there is a spelling error in the word 'latitude'. Let me know if you want any help fixing it so the tests will pass 😄

Comment (dhrubesh) [reply _pins/dhrubesh.json]: Oops :p I am unable to find, how to edit it can you guide me with it?

Comment (brianamarie) [reply _pins/dhrubesh.json]: Sure thing, @dhrubesh! There are three resources for you to look at that may help. 
- [Click here for some specific ideas](https://services.github.com/on-demand/intro-to-github/edit-files-on-github)
- Or [watch this video to get some help](https://vimeo.com/201011288)
- Here are step by step instructions. First, click on the 'files changed' tab in this pull request. Then, follow the directions in [this help article that walks through the steps of how to edit a file on GitHub.com](https://help.github.com/articles/editing-files-in-your-repository/)

Comment (dhrubesh) [reply _pins/dhrubesh.json]: Hey I changed it, but it still shows :
image

Comment (brianamarie) [reply _pins/dhrubesh.json]: I think you may not have committed the change, because when I look at the file I still am seeing 'lattitude'.

Comment (dhrubesh) [reply _pins/dhrubesh.json]: image

Comment (dhrubesh) [reply _pins/dhrubesh.json]: I am not getting the commit changes options: 
image

Comment (brianamarie) [reply _pins/dhrubesh.json]: Ah I see @dhrubesh. I don't think you are currently on your branch. Within this pull request. are you able to see the 'files changed' option at the top of the pull request? Then, can you edit from there? I will check your collaborator status to make sure that isn't the problem.

**Comment by brianamarie** [comment]
@dhrubesh If you click this link you should be able to commit directly: https://github.com/githubschool/open-enrollment-classes-introduction-to-github/invitations

Thank you for your patience in troubleshooting! 😄

**Comment by brianamarie** [comment]
Yay! Great job, @dhrubesh!

**Comment by dhrubesh** [comment]
Hey Thanks :) I am just confused about one thing, how do I delete this branch?

**Comment by brianamarie** [comment]
Since there isn't a button here to delete the branch, if you click the `code` tab near the top of the repository, then click `branches`, you should be able to find your branch there to delete.

**Pull request merged**